### ISSUE TITLE
[Accessbility]A11y_Java Tooling for IntelliJ/Eclipse - New Feature_HDInsight Library Settings_Screen Reader: Narrator is not announcing the role as link for the controls under "Details" expandable button

### ISSUE THREAD

**Issue by Manisha370** [microsoft/azure-tools-for-java#3570]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-NewFeature; #MAS1.3.1; #ScreenReader; #C+E-H; #JavatoolingforIntelliJ/Eclipse; #Narrator

**Environment Details:**
Eclipse IDE for Enterprise Java Developers
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:**
OS: Windows 10 Enterprise (Version 1903, Build: 18362.356)
Screen Reader: Narrator
MAS Violated: MAS1.3.1

**Repro Steps:**
1.Turn on Screen Reader
2.Open Java EE - Eclipse application
3.Install 'Azure Toolkit for Eclipse' from Marketplace
4.Open Eclipse and install  the Scala plug-in 
5.Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
6.Tab till File-->New-->Project and press enter
7. Tab till HDInsight and expand it. Select 'Spark on HDInsight (Scala)'. Tab till Next and press enter
8.Enter the Project name & tab till "Next" button and press enter
9.Tab till "Create new source folder", "Link additional source", "Add project 'HTG' to build path" links
10.Verify whether screen reader is announcing the role as link or not

**Actual Result:**
Narrator is not announcing the role as link for the controls "Create new source folder", "Link additional source", "Add project 'HTG' to build path"

**Expected Result:**
Narrator should announce the role as link along with the link names "Create new source folder", "Link additional source", "Add project 'HTG' to build path"

**User Impact:**
Screen Reader users will find difficulty in understanding the control-type, if role is not read 

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

[Bug3_Narrator is not announcing the role as links for the controls under Details expandable button.pptx](https://github.com/microsoft/azure-tools-for-java/files/3635790/Bug3_Narrator.is.not.announcing.the.role.as.links.for.the.controls.under.Details.expandable.button.pptx)
![Bug3_NVDA is announcing the role as link for the controls under details button](https://user-images.githubusercontent.com/51435937/65594999-6a788700-dfb1-11e9-95f6-fafa88bb2ffb.png)

**Comment by t-rufang** [comment]
This feature should be supported by Eclipse

**Comment by Alexandria-R** [comment]
This has been approved to close by SME.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749736
